src/worker/exporters/node_exporter.py

```diff
@@ -298,19 +303,22 @@ def init_config(job_id, port=None):
         config['command']['xid_error'] = "sudo grep 'NVRM: Xid' /var/log/syslog"
         init_nvidia_config()
         init_ib_config()
+
```

Comment:
Just for formatting maybe you don't need a leading space before the elif? I don't think there is generally a blank line before else other places in the code.

Reply:
sure let me address

Reply:
addressed